docs/getting_started/execution-modes.rst

```diff
@@ -159,6 +164,38 @@ Example DAG:
             "secrets": [postgres_password_secret],
         },
     )
+AWS_EKS
+----------
+
+The ``aws_eks`` approach is very similar to the ``kubernetes`` approach, but it is specifically designed to run on AWS EKS clusters.
+It uses the `EKSPodOperator <https://airflow.apache.org/docs/apache-airflow-providers-amazon/2.2.0/operators/eks.html#perform-a-task-on-an-amazon-eks-cluster>`_
```

Comment:
```suggestion
It uses the `EKSPodOperator <https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/operators/eks.html#perform-a-task-on-an-amazon-eks-cluster>`_
```

Reply:
Stable would point to latest and it may change. I think better to tag it to a specific version and also would denote that it works fine according to that version. Since we don't allow latest versions yet, I think 8.19.0 is a good candidate